guides/source/active_record_basics.md

```diff
@@ -586,8 +586,7 @@ unique, is not already in the database, follows a specific format, and many
 more.
 
 Methods like `save`, `create` and `update` validate a model before persisting it
-to the database. When a model is invalid these methods return `false` and no
-database operations are performed. All of these methods have a bang counterpart
+to the database. If the model is invalid, the `save` and `update` methods return `false`, and no database operations are performed. However, the `create` method does not return `false`; instead, it returns the ActiveRecord object, which can be checked for errors.  All of these methods have a bang counterpart
```

Comment:
can you also split the sentence into multiple lines. please check character count of other sentences in each line.

Reply:
It might be unclear that "no database operations are performed" for `create` as well. So maybe separate that?
Also I think it's more clear to just explain what `create` returns instead of what it doesn't return.
```suggestion
to the database. If the model is invalid, no database operations are performed. In this case the `save` and `update` methods return `false` instead of `true`. The `create` method still returns the ActiveRecord object, which can be checked for errors. All of these methods have a bang counterpart
```

Reply:
Sounds good to me.

Reply:
Done.